.github/workflows/python-app.yml

```diff
@@ -0,0 +1,74 @@
+name: Build Status
+   
+on:
+   push:
+   pull_request:
+   schedule:
+     - cron: '0 0 * * 0' # weekly
+
+defaults:
+  run:
+    shell: bash
+
+jobs:
+  build:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ubuntu-latest]
+#        os: [ubuntu-latest, macos-latest, windows-latest]
+        python-version: [3.6, 3.7, 3.8]
+#        exclude:
+#          - os: macos-latest
+#            python-version: 3.8
+#          - os: windows-latest
+#            python-version: 3.6
+
+    env:
```

Comment:
Why are these necessary? If they are necessary, can we change the name to not be `TRAVIS_` ?

Reply:
cleaned this. it is important to import the files from github to the environment of work.